spec/models/manageiq/providers/openstack/cloud_manager/refresher_spec.rb

```diff
@@ -1,5 +1,64 @@
 describe ManageIQ::Providers::Openstack::CloudManager::Refresher do
+  include Spec::Support::EmsRefreshHelper
+
+  before(:each) do
+    host = YAML.load_file("config/secrets.yml").dig("test", "openstack", "hostname")
+    port = YAML.load_file("config/secrets.yml").dig("test", "openstack", "port")
+    username = YAML.load_file("config/secrets.yml").dig("test", "openstack", "userid")
+    password = YAML.load_file("config/secrets.yml").dig("test", "openstack", "password")
```

Comment:
So _here_ is where you can use `Rails.application.secrets.openstack` and that will handle loading the defaults if secrets.yml isn't present